tasks/gaia.js

```diff
@@ -53,7 +53,7 @@ async function initGenesis(
   nodeHomeDir
 ) {
   await makeExec(
-    `${nodeBinary} add-genesis-account ${address} 150stake,1000photino  --home ${nodeHomeDir}`
+    `${nodeBinary} add-genesis-account ${address} 200000000stake,1000photino  --home ${nodeHomeDir}`
```

Comment:
🤑

Reply:
richness is gooood, but this change happened because of an error in the creation under 100000000stake